tests/integration/API/post-file.test.ts

```diff
@@ -0,0 +1,64 @@
+/* tslint:disable:no-relative-imports */
+import { asyncPipe } from 'Helpers/asyncPipe'
+import * as fs from 'fs'
+import * as path from 'path'
+import { prop, identity, head, map } from 'ramda'
+import { describe } from 'riteway'
+
+import { createFileHelper } from '../../helpers/files'
+import { createIPFS } from '../../helpers/ipfs'
+import { runtimeId, setUpServerAndDb } from '../../helpers/utils'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+
+const ipfs = createIPFS()
+const fileHelper = createFileHelper({ port: NODE_PORT  })
+
+const getHash = prop('Hash')
```

Comment:
`Hash` should be lowercase, `hash`. The code should convert this from IPFS response.